target/linux/lantiq/image/Makefile

```diff
@@ -757,89 +814,105 @@ endif
 endef
 TARGET_DEVICES += lantiq_easy98000-nor
 
 define Device/lantiq_easy98000-nand
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY98000 Falcon Eval Board
+  DEVICE_VARIANT := NAND
   IMAGE_SIZE := 3904k
   DEVICE_DTS := EASY98000NAND
-  DEVICE_TITLE := EASY98000 - Lantiq Falcon Eval Board NAND
   DEVICE_PACKAGES := kmod-dm9000 kmod-i2c-lantiq kmod-eeprom-at24
 endef
 TARGET_DEVICES += lantiq_easy98000-nand
 
 define Device/lantiq_easy98000-sflash
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY98000 Falcon Eval Board
+  DEVICE_VARIANT := SFLASH
   IMAGE_SIZE := 7424k
   DEVICE_DTS := EASY98000SFLASH
-  DEVICE_TITLE := EASY98000 - Lantiq Falcon Eval Board SFLASH
   DEVICE_PACKAGES := kmod-dm9000 kmod-i2c-lantiq kmod-eeprom-at24
 endef
 TARGET_DEVICES += lantiq_easy98000-sflash
 
 define Device/lantiq_falcon-mdu
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := MDU Lantiq Falcon / VINAXdp MDU Board
   IMAGE_SIZE := 7424k
   DEVICE_DTS := FALCON-MDU
-  DEVICE_TITLE := MDU - Lantiq Falcon / VINAXdp MDU Board
 endef
 TARGET_DEVICES += lantiq_falcon-mdu
 
 define Device/lantiq_easy88388
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY88388 Falcon FTTDP8 Reference Board
   IMAGE_SIZE := 7424k
   DEVICE_DTS := EASY88388
-  DEVICE_TITLE := EASY88388 - Lantiq Falcon FTTDP8 Reference Board
 endef
 TARGET_DEVICES += lantiq_easy88388
 
 define Device/lantiq_easy88444
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := EASY88444 Falcon FTTdp G.FAST Reference Board
   IMAGE_SIZE := 7424k
   DEVICE_DTS := EASY88444
-  DEVICE_TITLE := EASY88444 - Lantiq Falcon FTTdp G.FAST Reference Board
 endef
 TARGET_DEVICES += lantiq_easy88444
 
 define Device/lantiq_falcon-sfp
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := Falcon SFP Stick
   IMAGE_SIZE := 7424k
   DEVICE_DTS := FALCON-SFP
-  DEVICE_TITLE := SFP - Lantiq Falcon SFP Stick
 endef
 TARGET_DEVICES += lantiq_falcon-sfp
 
 define Device/lantiq_easy98035synce
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := Falcon SFP Stick with Synchronous Ethernet (EASY98035SYNCE)
   IMAGE_SIZE := 7424k
   DEVICE_DTS := EASY98035SYNCE
-  DEVICE_TITLE := EASY98035SYNCE - Lantiq Falcon SFP Stick with Synchronous Ethernet
 endef
 TARGET_DEVICES += lantiq_easy98035synce
 
 define Device/lantiq_easy98035synce1588
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := Falcon SFP Stick with SyncE and IEEE1588 (EASY98035SYNCE1588)
   IMAGE_SIZE := 7424k
   DEVICE_DTS := EASY98035SYNCE1588
-  DEVICE_TITLE := EASY98035SYNCE1588 - Lantiq Falcon SFP Stick with SyncE and IEEE1588
 endef
 TARGET_DEVICES += lantiq_easy98035synce1588
 
 define Device/lantiq_easy98020
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := Falcon SFU Reference Board (EASY98020)
   IMAGE_SIZE := 7424k
   DEVICE_DTS := EASY98020
-  DEVICE_TITLE := EASY98020 - Lantiq Falcon SFU Reference Board
 endef
 TARGET_DEVICES += lantiq_easy98020
 
 define Device/lantiq_easy98020-v18
+  DEVICE_VENDOR := Lantiq
+  DEVICE_MODEL := Falcon SFU Reference Board (EASY98020V18)
```

Comment:
@adrianschmutzler is this the correct format?

Reply:
"Falcon SFU Reference Board (EASY98020)" without V18, as the latter is the version.
Here again we also need a device variant for the previous currently without.
What would be appropriate there (just "v1"?)? @dangowrt ?

Reply:
ok, removed the V18 so far.